mwdb/model/user.py

```diff
@@ -184,11 +187,12 @@ def _verify_token(token, fields, scope):
 
         return user_obj
```

Comment:
You can just return `data` or `data.get("provider")` here, so there is no need for doubled verification

```suggestion
        return user_obj, data.get("provider")
```